### ISSUE TITLE
Have you reach the hit ratio in the paper?

### ISSUE THREAD

**Issue by Shane-Yu** [RuidongZ/Deep_Matrix_Factorization_Models#2]

**Comment by Shane-Yu** [comment]
First, I have trained the model many times，but i didn't reach the best hit ratio as it in the paper.
The second problem is that maybe you lost the activate function and bias B in user_inpuit layer and item_input layer.

**Comment by RuidongZ** [comment]
No. It is not an official implement. There might be some tricks which authors don't mention.
